msgraph-developer-proxy/ProxyConfiguration.cs

```diff
@@ -1,12 +1,39 @@
 ﻿// Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
+using System.Runtime.Serialization;
 using System.Text.Json.Serialization;
 
 namespace Microsoft.Graph.DeveloperProxy;
 
+public enum LabelMode {
+    [EnumMember(Value = "text")]
+    Text,
+    [EnumMember(Value = "icon")]
+    Icon,
+    [EnumMember(Value = "nerdFont")]
+    NerdFont
+}
+
+public enum LogLevel {
+    [EnumMember(Value = "debug")]
+    Debug,
+    [EnumMember(Value = "info")]
+    Info,
+    [EnumMember(Value = "warn")]
+    Warn,
+    [EnumMember(Value = "error")]
+    Error
+}
+
 public class ProxyConfiguration {
     [JsonPropertyName("port")]
     public int Port { get; set; } = 8000;
+    [JsonPropertyName("labelMode")]
+    [JsonConverter(typeof(JsonStringEnumConverter))]
+    public LabelMode LabelMode { get; set; } = LabelMode.Text;
+    [JsonPropertyName("logLevel")]
```

Comment:
Should we expose the LogLevel and LabelMode as command line options too?

Reply:
logLevel would possibly be useful. I don't think labelMode is something folks would want to change on a per-execution basis.